### PR TITLE
Setup initial code structure with Custom Action Bar & icon

### PR DESCRIPTION
**Pull request by SachinRammoorthy**

### CODE REVIEW

**Comment by SachinRammoorthy** [comment]
@garg-anuj done

File: app/app.iml [hunk 1]
```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (garg-anuj) [inline app/app.iml]: I don't feel this file is really needed in version control

**Comment by SachinRammoorthy** [comment]
@garg-anuj done

**Comment by garg-anuj** [comment]
Thanks for work @SachinRammoorthy

**Pull request closed without merging**